src/js/lib/youtube.js

```diff
@@ -65,6 +69,50 @@ export function pimpYouTubePlayer(videoId, node, height, width, chapters) {
             });
         });
     });
+
+
+    function chapterTimer(youTubePlayer, playerTotalTime) {
+        let chapterCurrentProgress;
```

Comment:
yeah, at least it's encapsulated in a function but other than that didn't want to touch it!